examples/quickstart-tensorflow/pyproject.toml

```diff
@@ -1,16 +1,20 @@
 [build-system]
-requires = ["poetry-core>=1.4.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling"]
```

Comment:
Does that require an install?

Reply:
From my testing it doesn't seem like it... But does it work on all systems?

Reply:
> Does that require an install? 

I don't believe it does.

> But does it work on all systems?

What systems did you have in mind? My initial thought was that this is a simple example to allow others to build on. We could equivalently use only `requirements.txt` and avoid `pyproject.toml` altogether. Perhaps it's a matter of opinion? 

One possible benefit with `pyproject.toml` that I can see here is that after running `pip install .` in each example, we effectively build a package (perhaps in an atypical way). This means that when we query `pip show {name-of-example}`, it will display the Flower authors and other metadata.